src/utils/chain.ts

```diff
@@ -0,0 +1,8 @@
+import { treatNetwork } from '../constants'
+import { store } from '../store'
+
+export function getNetworkAndProtocol(): [string, string] {
```

Comment:
nitpick: this could be a bit more concise
```javascript
const { network, chain } = store.getState().network
return [treatNetwork(network), chain]
```